### PR TITLE
Add process app to euth

### PR DESCRIPTION
**Pull request by slomo**

### CODE REVIEW

File: euth/process/util.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+from django.conf.urls import url
+
+
+def processUrl(suburl, views, **kwargs):
+    return url(r'^(?P<process_name>\w+)/' + suburl, views, **kwargs)
```
Comment (MagdaN) [inline euth/process/util.py]: This file is not used. Should it be removed?

File: euth/process/views.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    phases = process.phase_set.order_by('order')
+    context = {
+        'process': process,
+        'phases': phases,
+        'breadcrumbs': [
+            ('/', reverse('process-listing')),
+            (process.title, None),
+        ]
+    }
+    return render(request, 'process/detail.html', context)
```
Comment (MagdaN) [inline euth/process/views.py]: Maybe this could be done with generic modelviews?

File: euth/process/models.py [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+        ParticipationModule,
+        models.CASCADE,
+    )
+
+    date_start = models.DateTimeField(blank=True, null=True)
+    date_end = models.DateTimeField(blank=True, null=True)
+
+    def __str__(self):
+        return '{} phase of process «{}»'.format(
+            quantityStr(self.order), self.process.name)
```
Comment (MagdaN) [inline euth/process/models.py]: We need to add a validation to the phase checking that there is no other overlapping phase.

**Comment by slomo** [comment]
This code as been superseded by newer developments. Instead we will have pull-request for the projects app.

**Pull request closed without merging**